intranet/apps/dataimport/management/commands/import_eighth.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from intranet.apps.eighth.models import EighthActivity, EighthRoom, EighthSponsor
+
+
+class Command(BaseCommand):
+    help = "Import Fake Activities"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("data_fname")
+
+    def handle(self, *args, **kwargs):
+        try:
+            with open(kwargs["data_fname"]) as f_obj:
+                data = json.load(f_obj)
+        except OSError as ex:
+            raise CommandError(str(ex)) from ex
+
+        for activity in data["activities"]:
+            name = activity["Name"].strip()
+            description = activity["Description"].strip()
+            sponsor_first_name, sponsor_last_name = activity["Sponsor"].strip().split()
+            room_num = activity["Room Number"]
+            capacity = activity["Capacity"]
+            date = activity["Date"].strip()
+            classification = activity["Classification"]
```

Comment:
Does the `Classification` column not support multiple classifications? A lot of real activities have multiple classifications such as Restricted & Sticky together. It may be better to split up the classifications into Sticky, Presign, Oneaday, Special, Administrative, & Restricted columns. In your initial generation script, you can then pass `Y` or `N` (or equivalent classifications)